.gitignore

```diff
@@ -5,3 +5,7 @@
 docs/default.profraw
 docs/_site
 docs/.jekyll-metadata
+
+# Vim
```

Comment:
This can be reverted I merged your separate PR.

Reply:
Done.